smoketest/graphqloncallassignments_test.go

```diff
@@ -37,454 +56,833 @@ func TestGraphQLOnCallAssignments(t *testing.T) {
-			escalation_policy_steps: [{escalation_policy_id: "ep", delay_minutes: 1, targets: [{target_type: schedule, target_id: "s" }] }]
-			services: [{id_placeholder: "svc", description: "ok", name: "generatedA", escalation_policy_id: "ep"},{description: "ok", name: "generatedB", escalation_policy_id: "ep"}]
-			schedules: [{id_placeholder: "s", time_zone: "UTC", name: "generatedA", description: "1"}]
-			schedule_rules: [{target:{target_type:rotation, target_id:"rot"}, start:"00:00", end:"23:59", schedule_id: "s", sunday: true, monday:true, tuesday:true, wednesday: true, thursday: true, friday: true, saturday: true}]
-			rotations: [{id_placeholder: "rot", time_zone: "UTC", shift_length: 1, type: weekly, start: "2006-01-02T15:04:05Z", name: "generatedA", description: "1"}]
-			rotation_participants: [{rotation_id: "rot", user_id: "u2"}]
-			user_overrides: [{add_user_id: "u1", remove_user_id: "u2", start_time: "1006-01-02T15:04:05Z", end_time: "4006-01-02T15:04:05Z", target_type: schedule, target_id: "s"}]
-		`,
-		[]resolver.OnCallAssignment{
-			{ServiceName: "generatedA", EPName: "generatedA", ScheduleName: "generatedA", Level: 0, IsActive: true},
-			{ServiceName: "generatedB", EPName: "generatedA", ScheduleName: "generatedA", Level: 0, IsActive: true},
-		},
-		[]resolver.OnCallAssignment{
-			{ServiceName: "generatedA", EPName: "generatedA", ScheduleName: "generatedA", Level: 0, IsActive: false},
-			{ServiceName: "generatedB", EPName: "generatedA", ScheduleName: "generatedA", Level: 0, IsActive: false},
-		},
+	// User EP Schedule Replace Rotation Override Double Service
+	check("User EP Schedule Replace Rotation Override Double Service", fmt.Sprintf(`
+		mutation {
+		alias0: createService(input: { name: "{{name "svc1"}}", escalationPolicyID: "%s" }) {
+			id
+		}
+		alias1: createService(input: { name: "{{name "svc2"}}", escalationPolicyID: "%s" }) {
+			id
+		}
+		createEscalationPolicyStep(
+			input: {
+				escalationPolicyID: "%s"
+				delayMinutes: 1
+				newSchedule: {
+					name: "{{name "sched"}}"
+					timeZone: "UTC"
+					targets: [
+						{
+							newRotation: {
+								name: "{{name "rot"}}"
+								type: weekly
+								shiftLength: 1
+								timeZone: "UTC"
+								start: "2006-01-02T15:04:05Z"
+								userIDs: ["{{userID "joe"}}"]
+							}
+							rules: [
+								{
+									start: "00:00"
+									end: "23:59"
+									weekdayFilter: [true, true, true, true, true, true, true]
+								}
+							]
+						}
+					]
+					newUserOverrides: [
+						{
+							addUserID: "{{userID "bob"}}"
+							removeUserID: "{{userID "joe"}}"
+							start: "1006-01-02T15:04:05Z"
+							end: "4006-01-02T15:04:05Z"
+						}
+					]
+				}
+			}
+		) {
+			id
+		}
+	}`, h.UUID("eid"), h.UUID("eid"), h.UUID("eid")),
+		[]onCallAssertion{{Service: "svc1", EPName: "esc policy", StepNumber: 0, User: "bob"}, {Service: "svc2", EPName: "esc policy", StepNumber: 0, User: "bob"}},
```

Comment:
Put one per line to make it easier to read
```suggestion
		[]onCallAssertion{
			{Service: "svc1", EPName: "esc policy", StepNumber: 0, User: "bob"},
			{Service: "svc2", EPName: "esc policy", StepNumber: 0, User: "bob"},
		},
```